### ISSUE TITLE
Hyphen unicode character not displaying

### ISSUE THREAD

**Issue by zackslash** [jeffvli/feishin#151]
## Expected Behavior

Band or song names that include the Hyphen (U+2010) character display correctly.

## Current Behavior

Any text that includes the Unicode hyphen character displays an unmapped font symbol.

Example string 'Static‐X':
![Screenshot 2023-06-19 at 13 45 53](https://github.com/jeffvli/feishin/assets/2040617/0eb31032-0fe7-484e-8845-141a336a37d7)

![Screenshot 2023-06-19 at 13 45 41](https://github.com/jeffvli/feishin/assets/2040617/b5fad5bc-60f1-4bc3-8e6b-68ed709b7087)

## Steps to Reproduce

View any text that uses the Hyphen ([U+2010](https://unicodeplus.com/U+2010)) character (‐) 
Note: not to be confused with the similar 'Hyphen-Minus' symbol ([U+002D](https://unicodeplus.com/U+002D)) (-) as this currently displays correctly.

## Context

Any artist or song that includes the hyphen symbol does not display correctly.

## Solution

EDIT: see reply, looks like the solution is to upgrade electron version.

## Your Environment
- Application version (e.g. v0.1.0) : v0.2.0
- Operating System and version (e.g. Windows 10) : Mac OS Ventura 13.4
- Server and version (e.g. Navidrome v0.48.0) : Navidrome
- Node version (if developing locally) : v20.3.0

**Comment by zackslash** [comment]
After further investigation of this issue I think it is related to [this bug](https://bugs.chromium.org/p/chromium/issues/detail?id=1267606
) in chromium. It looks like it was fixed in chromium 111, I think that corrosponds to electron versions > 24.

I tested upgrading the Feishin electron version to 24.5.1 and this issue does dissappear.

![Screenshot 2023-06-21 at 21 21 03](https://github.com/jeffvli/feishin/assets/2040617/f8df35b9-a33f-45f2-97db-56b225513bd1)

So I guess the next question is: can the electron version be upgraded? 😃
